### PR TITLE
Add "Become a Committer" page with guidelines from the PMC

### PR DESCRIPTION
**Pull request by kennknowles**
The PMC has reached consensus on these committer guidelines. I also streamlined the "Team" page to just be a roster.

### CODE REVIEW

**Comment by kennknowles** [comment]
Pulling out staging link: http://apache-beam-website-pull-requests.storage.googleapis.com/434/index.html

**Comment by kennknowles** [comment]
I had some issues with the sidebar when there is a link in a title, so I moved the links out of the title. There is also an issue with the sidebar on the Team page, but that is already a problem on the live site.

**Comment by davorbonaci** [comment]
I'd suggest to get rid of the Team page altogether, and simply link/integrate/embed some of the following resources:
* https://projects.apache.org/committee.html?beam
* https://people.apache.org/phonebook.html?ctte=beam
* https://people.apache.org/phonebook.html?unix=beam

**Comment by kennknowles** [comment]
Nice! I really like the first link as a summary. It might take a little thought how to make it discoverable in the right place, so I filed https://issues.apache.org/jira/browse/BEAM-4234.

**Comment by melap** [comment]
One option would be to leave the Team entry in the left nav, but add the little external icon onto it (like the Java and Python API reference entries have on the SDKs page) and link directly to that page.

File: src/contribute/become-a-committer.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+See the [ASF documentation](https://www.apache.org/dev/committers.html#committer-responsibilities).
+
+ - They help create a product that will outlive the interest of any particular
+   volunteer (including themselves)
+ - They grow and maintain the health of the Apache community
+ - They help out with surrounding work, such as the website & documentation
+ - They help users
+ - They can be trusted to decide when code is ready for release, or when to ask
+   someone else to make the judgment
+ - `code contributors` They can be trusted to decide when to merge code, or
```
Comment (melap) [inline src/contribute/become-a-committer.md]: just a minor pondering: I wonder if it'd be clearer to put code contributors in parens (and optionally, perhaps use italics or no formatting at all)

Comment (kennknowles) [reply src/contribute/become-a-committer.md]: Tried it out, with some sub-bullets.

**Comment by kennknowles** [comment]
@asfgit merge

**Comment by kennknowles** [comment]
Great. Trying my luck - some of our dependent sites seem to be down or modified.

**Comment by kennknowles** [comment]
retest this please

**Comment by kennknowles** [comment]
Actually IIRC the bot does not handle new files anyhow. Going manual.

**Pull request merged**